pkg/landscaper/installations/subinstallations/helper_test.go

```diff
@@ -161,7 +161,7 @@ var _ = Describe("SubinstallationsHelper", func() {
 			sortInstallationTemplatesAlphabetically(tmpls)
 			_, err := subinstallations.OrderInstallationTemplates(tmpls)
 			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).To(Equal("Op: EnsureNestedInstallations - Reason: OrderNestedInstallationTemplates - Message: The following cyclic dependencies have been found in the nested installation templates: {c -[e_data]-> e -[d_data]-> d -[c_data]-> c}"))
+			Expect(err.Error()).To(Equal("Op: EnsureNestedInstallations - Reason: OrderNestedInstallationTemplates - Message: The following cyclic dependencies have been found in the nested installation templates: {c -[e_data]-> e -[d_mapping]-> d -[c_target]-> c}"))
```

Comment:
@Diaphteiros Are these changes also related to the golang version update?
I'm just curious, whether this slipped in by accident.

Reply:
No, this was on purpose ... it has been wrong the whole time, but I just didn't notice until I updated the linter and got linting errors, which, after I fixed them, revealed that the test was wrong.